lib/client/api.go

```diff
@@ -488,7 +491,7 @@ type Config struct {
 	SSHLogDir string
 
 	// MFAPromptConstructor is a custom MFA prompt constructor to use when prompting for MFA.
-	MFAPromptConstructor func(cfg *libmfa.PromptConfig) mfa.Prompt
+	MFAPromptConstructor func(cfg libmfa.PromptConfig) mfa.Prompt
```

Comment:
Why drop the pointer?

Reply:
Because we end up dereferencing it, it doesn't seem as safe. WDYT?

Reply:
Depending on how much the type grows I think the pointer is better. We can check for nil before de-referencing. That said you are way deeper here than I am.

Reply:
Reverted for now